orchestrator/process/utils.go

```diff
@@ -25,29 +25,37 @@ func buildStagedProcessesList(managerConfig config.ProcessesManagerConfig) []pro
 	var stagedProcessesList []process
 
 	for _, group := range managerConfig.ProcessGroups {
-		groupArgs := getProcessGroupArgs(group.Script, group.ArgPrefix, group.ArgAssign, group.Args)
+		groupArgs := getProcessArgs(group.Script, group.ArgPrefix, group.ArgAssign, group.Args.GroupArgs)
 		groupInstance := newProcessGroup(group.Name, group.Replicas, group.Command, groupArgs)
 
 		for i := 0; i < group.Replicas; i++ {
-			stagedProcessesList = append(stagedProcessesList, newProcess(groupInstance))
+			processArgs := []string{}
+			if i < len(group.Args.IndividualArgs) {
```

Comment:
Why do we need the condition? Wouldn't `getProcessArgs` return an empty array?

Reply:
if there is a process that wasn't given an individual argument, then group.Args.IndividualArgs will go out of bound